build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/webpackTask.ts

```diff
@@ -56,22 +51,60 @@ export class WebpackTask extends LeafWithDoneFileTask {
 			}
 
 			return JSON.stringify(content);
-		} catch {
+		} catch (e) {
+			this.traceExec(`error generating done file content ${e}`);
 			return undefined;
 		}
 	}
 
 	private get configFileFullPath() {
 		// TODO: parse the command line for real, split space for now.
 		const args = this.command.split(" ");
-		let configFile = "webpack.config.js";
-		for (let i = 0; i < args.length; i++) {
+		for (let i = 1; i < args.length; i++) {
 			if (args[i] === "--config" && i + 1 < args.length) {
-				configFile = args[i + 1];
-				break;
+				return path.join(this.package.directory, args[i + 1]);
 			}
 		}
 
-		return path.join(this.package.directory, configFile);
+		return this.getDefaultConfigFile();
+	}
+
+	private getDefaultConfigFile() {
+		const defaultConfigFileNames = [
+			"webpack.config",
+			".webpack/webpack.config",
+			".webpack/webpackfile",
+		];
+		// TODO: webpack support more default config file extensions.  Just implement the ones that we use.
+		const defaultConfigExtensions = [".js", ".cjs"];
```

Comment:
Might be worth including mjs.

Reply:
yeah... but that needs another "import" style.  Since there isn't usage, I don't want to include that yet.